api/v1beta1/policyautomation_types.go

```diff
@@ -68,6 +72,12 @@ type PolicyAutomationList struct {
 	Items           []PolicyAutomation `json:"items"`
 }
 
+// PolicyAutomation event time on each cluster
+type ClusterEventTime struct {
+	AutomationStartTime string `json:"automationStartTime"`
```

Comment:
Could you please add comments to these fields so that they end up documenting the fields in the generated CRD?